Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -598,5 +596,83 @@ public void ProcessDelistings(Delistings delistings)
                 }
             }
         }
+
+        private void RemoveOrders(List<Order> orders, OrderStatus orderStatus, string message = "")
+        {
+            for (var i = 0; i < orders.Count; i++)
+            {
+                RemoveOrder(orders[i], orderStatus, message);
+            }
+        }
+
+        private void RemoveOrder(Order order, OrderStatus orderStatus, string message = "")
+        {
+            OnOrderEvent(new OrderEvent(order, Algorithm.UtcTime, OrderFee.Zero, message) { Status = orderStatus });
+
+            _pending.TryRemove(order.Id, out var _);
+        }
```

Comment:
Cool! so now I think we can remove `RemoveOrder` and call `OnOrderEvents` in `RemoveOrders` ?